### PR TITLE
Add installation guides for Quest devices.

### PR DESCRIPTION
**Pull request by IMB11**

### CODE REVIEW

**Review by PangwenE — APPROVED** [review]

File: installation/quest/download.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
 # Downloading Noodle Extensions
 
-[TBD] cal117 is working on this page.
+You can download Noodle Extensions for quest from the following sources:
+
+- [noodleextensions.com](https://noodleextensions.com/#download)
+- [Beat Saber Modding Group](https://discord.gg/beatsabermods)
+- [GitHub](https://github.com/StackDoubleFlow/NoodleExtensions/releases/latest)
+
+Make sure to download the file that has the `.qmod` file extension - `.so` and `.zip` are invalid.
```
Comment (DefDoof) [inline installation/quest/download.md]: Maybe add "or any other file types are invalid" at the end because test review :) 💯

**Review by DefDoof — COMMENTED** [review]
Hi? This workie workie?

File: installation/quest/download.md (hunk 1 above)
Comment (IMB11) [reply installation/quest/download.md]: Okay :)

**Review by DefDoof — APPROVED** [review]
okay this time workie workie?

File: installation/quest/download.md (hunk 1 above)
Comment (IMB11) [reply installation/quest/download.md]:
```suggestion
Make sure to download the file that has the `.qmod` file extension - any other file types are invalid.
```

Comment (IMB11) [reply installation/quest/download.md]: Commited

**Pull request merged**